src/Take.Elephant.Sql/MapStorageBase.cs

```diff
@@ -53,7 +53,7 @@ protected MapStorageBase(IDatabaseDriver databaseDriver, string connectionString
                 int totalCount = 0;
                 if (FetchQueryResultTotal)
                 {
-                    using (var countCommand = connection.CreateSelectCountCommand(DatabaseDriver, Table.Schema, Table.Name, filter.Where, filter.FilterValues))
+                    using (var countCommand = connection.CreateSelectCountCommand(DatabaseDriver, Table.Schema, Table.Name, Table.Columns, filter.Where, filter.FilterValues))
```

Comment:
I think all these methods receiving Table.Columns can just receive the ITable, instead of some parts of it. We are getting into a code smell (https://refactoring.guru/smells/long-parameter-list)